cmd/metacache-manager.go

```diff
@@ -43,6 +43,7 @@ type metacacheManager struct {
 }
 
 const metacacheManagerTransientBucket = "**transient**"
+const metacacheMaxEntries = 10000
```

Comment:
5000 is enough?

Reply:
Well, its just a number, right. :)

10000 sounded excessive. But so does 5K :)